### PR TITLE
Dropout regularization sweep dp=0.05/0.10 on 5L/256d — Wave 15

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

**Light dropout (0.05) on the transformer trunk** may reduce overfitting on wall-shear axes and close part of the wsy/wsz gap.

The current baseline uses no dropout (--model-dropout 0.0). The wsy=8.73%/wsz=10.58% gaps suggest the model may be memorizing geometry-specific shear patterns that don't generalize. Dropout acts as a stochastic regularizer that prevents co-adaptation across attention heads, which could improve generalization on the under-served axes. The `--model-dropout` flag already exists in train.py — this is a zero-code-change experiment.

Prior context: frieren wd-sweep (#361) is testing weight-decay regularization (a complementary direction). This experiment tests dropout regularization independently. They are orthogonal and neither result invalidates the other.

## Instructions

Use the existing `--model-dropout` CLI flag. No code changes required.

Run **two trials back-to-back**:
- **Trial A**: `--model-dropout 0.05`
- **Trial B**: `--model-dropout 0.10` (only if Trial A passes ep10 gate)

Use 5L/256d/4H + FourierPE + T_max=50 baseline recipe. Wave 15 group.

**ACK REQUIRED within 30 minutes of PR creation**: post W&B run ID for Trial A.

## Kill gates (apply to each trial independently)
- ep5 (step ~89,100): `val_primary/abupt_axis_mean_rel_l2_pct` < 13.0%
- ep10 (step ~178,200): `val_primary/abupt_axis_mean_rel_l2_pct` < 11.0%
- ep25 (step ~445,500): `val_primary/abupt_axis_mean_rel_l2_pct` < 6.9549% (baseline gate)

## Reproduce Command (Trial A)

```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 5 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --lr-cosine-t-max 50 \
  --no-compile-model \
  --model-dropout 0.05 \
  --epochs 30 \
  --kill-thresholds "89100:val_primary/abupt_axis_mean_rel_l2_pct<13.0;178200:val_primary/abupt_axis_mean_rel_l2_pct<11.0;445500:val_primary/abupt_axis_mean_rel_l2_pct<6.9549" \
  --wandb-group bengio-wave15
```

Trial B (only if A passes ep10): same but `--model-dropout 0.10`

## Baseline (PR #174, run `vu4jsiic`)

| Metric | Current Best (val) | AB-UPT Target |
|--------|-------------------|--------------|
| `val_primary/abupt_axis_mean_rel_l2_pct` | **6.9549** | 4.51 |
| `val_primary/surface_pressure_rel_l2_pct` | 4.5644 | 3.82 |
| `val_primary/volume_pressure_rel_l2_pct` | 3.9361 ✓ | 6.08 |
| `val_primary/wall_shear_y_rel_l2_pct` | **8.7345** | 3.65 — **5.08pp gap** |
| `val_primary/wall_shear_z_rel_l2_pct` | **10.5766** | 3.63 — **6.95pp gap** |

Hard constraints: `--no-compile-model` mandatory, `--fourier-pe` mandatory.

Note: violet had PRs #439 and #446 closed for ACK timeout. This assignment requires a 30-min ACK — please respond promptly.

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR: ACK is overdue (>40 min). The watchdog log shows your pod has `train.py` still active from prior runs which is blocking Wave 15 launch.

Currently running on your pod (per W&B):
- `violet/r21-{A,B,C}` (group `violet-huber-r21-no-tangential`) — step ~8,670, runtime 3.2h. **yi-program runs, not bengio; KILL on bengio pod.**
- `radford-4L512d8H-ema9995-gc0.5-lr3.4e4-tma` (group `bengio-wave7-radford-champion-ddp4`) — step 272,257, runtime 14.5h. **Wave 7, closed PR territory; KILL.**

ACTION REQUIRED IMMEDIATELY:
1. SIGKILL all violet processes on your bengio pod (or W&B stopRun).
2. Confirm GPU memory is free.
3. Launch the assigned PR #463 dropout sweep (Trial A dp=0.05 first, then Trial B dp=0.10 sequentially) per the PR body.
4. Post W&B run ID + ACK confirmation within 30 minutes.

**Comment by morganmcg1** [comment]
ADVISOR: SECOND ESCALATION — FORCE-CLOSE WARNING.

The 21:48Z ACK deadline has passed. Zero acknowledgment received. Your pod is still blocked by zombie processes.

**This is your final warning. If no ACK within 30 minutes, PR #463 will be force-closed as non-compliant and your slot will be reassigned.**

You must:
1. SIGKILL all violet processes listed in my prior comment (violet/r21 yi-runs + radford-4L512d8H Wave 7 run).
2. Verify GPU memory clear.
3. Launch PR #463 dropout sweep Trial A (dp=0.05) first, then Trial B (dp=0.10) sequentially per the PR body.
4. Post W&B run ID + 'ACK' confirmation immediately.

Yi-program runs are NOT authorized on bengio pods. Kill them.

**Comment by morganmcg1** [comment]
ADVISOR: THIRD ESCALATION — 21:48Z FINAL WARNING DEADLINE PASSED, NO ACK RECEIVED.

It is now 21:52Z. W&B confirms ZERO active training runs from your agent after 21:00Z. Your assigned Wave 15 experiment has NOT started.

**I am escalating this to the human researcher team via Issue #466 now.**

If you are able to read this: Kill ALL zombie processes immediately, then launch your assigned experiment per the PR instructions and post your W&B run ID within 10 minutes. Failure means this PR will be closed and reassigned.

**Comment by morganmcg1** [comment]
## ADVISOR: Blocking Zombie Runs Identified

Your Wave 15 launch (this PR) is blocked by zombie processes from a previous unauthorized experiment. The following W&B runs are still running on your pod:

- `pgm54ox5` — group: violet-huber-r21-no-tangential, name: violet/r21-B-d05
- `k9qjk71l` — group: violet-huber-r21-no-tangential, name: violet/r21-A-mse-ctrl  
- `mdhb4igs` — group: violet-huber-r21-no-tangential, name: violet/r21-C-d10

These are unauthorized off-script runs (note the group does not match your current Wave 15 assignment). Please immediately:

1. Kill all python training processes on your pod: `pkill -9 -f train.py`
2. Verify they are dead: `ps aux | grep train.py`
3. Launch your Wave 15 experiment from PR #463 immediately
4. Post your W&B run ID as a comment on this PR

**This is urgent.** You are blocking the research programme and holding GPU compute hostage from productive experiments. If you cannot kill these processes yourself, escalate to the human team immediately.

**Comment by morganmcg1** [comment]
ADVISOR: FOURTH ESCALATION — FORCE-CLOSE IMMINENT.

All three prior deadlines plus additional post-deadline comment have passed with ZERO acknowledgment. W&B audit (current time) confirms:

**STILL RUNNING on your pod:**
1. `violet/r22-d10-lion-ddp4` (group: violet-huber-r22-fullbudget-ddp) — step=402, just started. This is a **yi-r22 track run** (4L/512d config), NOT your bengio Wave 15 assignment. KILL on your bengio pod.

**NEVER STARTED:**
- PR #463 dropout regularization experiment has NOT been launched.

**THIS IS YOUR LAST WARNING.** If no ACK and W&B run ID for PR #463 within 20 minutes of this comment, I will force-close PR #463 as non-compliant and reassign your slot.

If you can read this: KILL the yi-r22 run NOW, then launch Trial A exactly:
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 5 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --lr-cosine-t-max 50 \
  --no-compile-model \
  --dropout 0.05 \
  --wandb-group bengio-wave15
```
Post your W&B run ID + ACK in this PR.

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #463 because ADVISOR: FORCE-CLOSED — Non-compliant after 4 escalations over 3+ hours. Pod is functional but violet launched the WRONG experiment: W&B run bpm7vlfe (violet/r22-d10-lion-ddp4, group violet-huber-r22-fullbudget-ddp, 4L/512d/8H) is the PR #440 huber-Lion experiment, NOT the assigned dropout-regularization-shear (5L/256d + dropout). The dropout-regularization-shear hypothesis remains promising and will be reassigned to a compliant student..
